7.1 дз/7.1 дз/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Homework
+{
+
+    class Program
+    {
+
+        public bool CountNum(double m, double n)
+        {
+            if (m > 0 | n > 0)
+                return true;
+            else 
+                return false;
+        }
+
+        static void Main()
+        {
+            var pr = new Program();
+            Console.WriteLine("Введите два числа");
+            double m = Convert.ToDouble(Console.ReadLine());
+            double n = Convert.ToDouble(Console.ReadLine());
+            Console.WriteLine(pr.CountNum(m, n));
```

Comment:
И это  код с правильным (соответствующим заданию) типом данных и со статическим методом был бы проще.
Кроме того рекомендую не использовать без особой необходимости методы класса `Convert`. Используйте методы `Parse` соответствующих типов.
```suggestion
            Console.WriteLine("Введите два целых числа");
            var m = int.Parse(Console.ReadLine());
            var n = int.Parse(Console.ReadLine());
            Console.WriteLine(CountNum(m, n));
```